### PR TITLE
Updating author profile as per figma mockups

### PR DESCRIPTION
**Pull request by shailee-m**
## Fixes
Fixes https://github.com/creativecommons/project_creativecommons.org/issues/142  by @brylie 

## Description
Updated author profile as per figma mockups. Not overriding the vocabulary classes at the moment.

## Checklist
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

### CODE REVIEW

File: author.php [hunk 1]
```diff
@@ -1,22 +1,24 @@
 <?php
-	get_header();
-	$author_image = get_avatar( get_queried_object()->ID, 'squared' );
+get_header();
+$author_image_url = get_avatar_url(get_queried_object()->ID, 'squared');
```
Comment (brylie) [inline author.php:3]: Let's create some context variables for re-use throughout the template code.

```suggestion
$author_object = get_queried_object();
$author_meta = get_user_meta(get_queried_object()->ID);

$author_id = $author_object->ID;
$author_nicename = $author_object->user_nicename;
$author_first_name = $author_meta["first_name"];
$author_position = $author_meta["cc_position"];
$author_description = $author_meta["description"];
$author_image_url = get_avatar_url($author_id, 'squared');
```

File: author.php [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
 				</div>
 			<?php endif; ?>
 			<div class="column">
-				<h2><?php echo CC_Site::page_title(); ?></h2>
-				<?php
-				if ( function_exists( 'yoast_breadcrumb' ) ) {
-					yoast_breadcrumb( '<p id="breadcrumbs">', '</p>' );
-				}
-				?>
+				<h2><?php echo get_queried_object()->user_nicename ?></h2>
```
Comment (brylie) [inline author.php:19]: We can make this code a bit more clear with the previously suggested context variables.
```suggestion
				<h2><?php echo $author_nicename ?></h2>
```

File: author.php [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
+				<h4><?php echo join(", ", get_user_meta(get_queried_object()->ID)["cc_position"]) ?></h4>
+				<p class="mt-2"><?php echo nl2br(join(", ", get_user_meta(get_queried_object()->ID)["description"])) ?></p>
```
Comment (brylie) [inline author.php:21]: - We can make this code a bit more clear with the previously suggested context variables.
- Is the description field in the author meta stored as a list? I'm just trying to understand the use of `join`.

```suggestion
				<p class="mt-2"><?php echo nl2br(join(", ", $author_description) ?></p>
```

File: author.php [hunk 4]
```diff
@@ -26,16 +28,34 @@
 		<div class="columns padding-vertical-larger">
 			<div class="column is-8">
 				<?php
-				if ( have_posts() ) :
-					while ( have_posts() ) :
+				if (have_posts()) :
+					echo "<h2 class='mb-2'>" . join(" ", get_user_meta(get_queried_object()->ID)["first_name"]) . "'s Posts</h2>";
```
Comment (brylie) [inline author.php:32]: - let's use the context variable for improved readability
- Why do we use `join` here?

```suggestion
					echo "<h2 class='mb-2'>" . join(" ", $author_first_name) . "'s Posts</h2>";
```

**Review by brylie — CHANGES_REQUESTED** [review]
This looks great overall.

I added a few minor suggestions for readability and a couple of open questions about using `join`.

**Comment by shailee-m** [comment]
@brylie Wordpress is accepting arrays in all keys which is why. We can also just use the first element in the array. I will update the code to do that in case

**Comment by shailee-m** [comment]
@brylie @annatuma Here is the look of the page after all the updates.
Untitled_ Dec 14, 2021 3_21 PM (2)

**Comment by brylie** [comment]
@shailee-m, how difficult do you think it might be to use a different image size here? From what I can tell, WordPress seems to be using the 'post-thumbnail` format returned by `the_post`. Is that correct?

I don't want the image format to be a blocking issue in this PR so I'm just asking about the difficulty. We can consider whether the image format needs to change and split that into a follow-up task if needed.

**Review by brylie — CHANGES_REQUESTED** [review]
This looks good and is ready to merge from what I can see.

The only change would be to resolve the conflict(s).

**Comment by shailee-m** [comment]
> This looks good and is ready to merge from what I can see.
> 
> The only change would be to resolve the conflict(s).

Resolved conflicts

**Review by brylie — APPROVED** [review]

**Pull request merged**